spec/index.html

```diff
@@ -271,10 +271,11 @@ <h3>RDF Vocabularies and Namespace IRIs</h3>
     <p>The <a>IRIs</a> in an <a>RDF vocabulary</a> often begin with
       a common substring known as a <dfn>namespace IRI</dfn>.
       Some namespace IRIs are associated by convention with a short name
-      known as a <dfn>namespace prefix</dfn>. Some examples:
+      known as a <dfn>namespace prefix</dfn>.
+      The table below provides some examples.
```

Comment:
```suggestion
      known as a <dfn>namespace prefix</dfn>.
```

The table caption no longer says "example" but has the more definitive "used in this specification".
As the caption text is immediately after this paragraph, just drop the "The Table ..."